javaslang/generator/Generator.scala

```diff
@@ -1558,6 +1569,19 @@ def generateTestClasses(): Unit = {
         """
       }
 
+      def genExtAliasTest(name: String, func: String, value: String, check: String): String = {
+        xs"""
+          @$test
+          public void should${name}() {
+              assertThat($func($value)).$check;
+          }
+        """
+      }
+
+      def genMediumAliasTest(name: String, func: String, value: String): String = genExtAliasTest(name + "ReturnNotNull", func, value, "isNotNull()")
```

Comment:
For one str concatenation an interpolated String looks like too much overhead. I would not do it... But the question is valid.